src/renderer.ts

```diff
@@ -419,9 +419,40 @@ const render = (format: Format, imageMappings: ImageMappings, excludeStyles = fa
             const { html, css: styles, js } = element;
 
             const atom = h(InteractiveAtom, { html, styles, js, format });
+            const fenced = `
+                <html>
+                    <head>
+                        <meta charset="utf-8">
+                        <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+                        <script>
+                            const fonts = [].slice.apply(window.parent.document.styleSheets)
+                                .filter(sheet => sheet.ownerNode.attributes?.getNamedItem('data-emotion-css')?.value?.includes('PageStyles'))
+                                .map(sheet => sheet.ownerNode.textContent)
+                                .join(' ')
+                            var css = document.createElement('style');
+                            css.textContent = fonts;
+                            document.head.appendChild(css);
+                        </script>
+                        <style>
+                            ${styles}
+                        </style>
+                    </head>
+                    <body>
+                        ${html}
+                        <script>
+                            ${js}
+                            function resize() {
```

Comment:
What does this do out of curiosity 🤔?

Reply:
It resizes the iframed interactive atom to the correct height. Otherwise it would default to the standard 150px iframe height